editor/src/clj/editor/workspace.clj

```diff
@@ -42,8 +43,17 @@ ordinary paths."
   (^File [workspace evaluation-context]
    (io/as-file (g/node-value workspace :root evaluation-context))))
 
+(defn- skip-first-char [^String path]
+  (.substring path 1 (count path)))
+
```

Comment:
You can get rid of this and use the built-in `subs` function.
https://clojure.github.io/clojure/clojure.core-api.html#clojure.core/subs